src/libraries/Silk.Net.Windowing/Abstract/Version32.cs

```diff
@@ -0,0 +1,70 @@
+﻿
+using System;
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+namespace Silk.NET.Core
+{
+    /// <summary>
+    /// A 32-bit version structure.
+    /// </summary>
+    public readonly struct Version32
```

Comment:
Wrong folder I think

Reply:
That one goes in core, doesn't it? My bad